src/main/resources/mamute.properties

```diff
@@ -14,8 +14,6 @@ vraptor.errorcontrol.date.joda.pattern = dd/MM/yyyy
 tags.splitter.regex = \\,
 # Char used to split tags inside the JSP forms. It must matchs with the tags.splitter.regex property
 tags.splitter.char = ,
-# Regex of allowed characters for tag names
-tags.sanitizer.regex = [a-zA-Z0-9-]
```

Comment:
We need to transliterate characters like čžš to czs, and only than we could url-encode non-transliteratable characters like # and +. I'll prepare something and commit it here.